src/DynamoInstall/Product.wxs

```diff
@@ -142,12 +141,6 @@
       </Component>
     </DirectoryRef>
 
-    <ComponentGroup Id="AddinGenerator">
```

Comment:
@sharadkjaiswal 
in the prebuild event, *DynamoAddinGenerator.exe* and *RevitAddinUtility.dll* are copied into the harvest folder